ykrt/src/lib.rs

```diff
@@ -6,8 +6,23 @@
 #![allow(clippy::type_complexity)]
 #![allow(clippy::new_without_default)]
 
+use std::env;
+
 mod location;
 pub(crate) mod mt;
 
 pub use self::location::Location;
 pub use self::mt::{HotThreshold, MT};
+
+#[cfg(feature = "jit_state_debug")]
+const JIT_STATE_ENV: &str = "YKD_PRINT_JITSTATE";
+
+/// Print select JIT events to stderr for testing/debugging purposes.
+#[cfg(feature = "jit_state_debug")]
+pub fn print_jit_state(state: &str) {
```

Comment:
This means we do an env var really frequently: I suggest we make this a global (like we do in `mt.rs` for the other thing). I also suspect this might better live in `mt.rs` too?